test/dns_dnscmd/dnscmd_test.rb

```diff
@@ -21,25 +21,33 @@ def setup
   end
 
   def test_create_a_record_with_longest_zone_match
-    Proxy::Dns::Dnscmd::Record.any_instance.expects(:execute).with('/RecordAdd bar.domain.local host.foo.bar.domain.local. A 192.168.33.33', anything).returns(true)
+    Proxy::Dns::Dnscmd::Record.any_instance.expects(:execute).with('/RecordAdd', 'bar.domain.local', 'host.foo.bar.domain.local.', 'A', '192.168.33.33', anything).returns(true)
```

Comment:
Please change all tests in the following way:

```suggestion
    @server.expects(:popen3).with('c:\Windows\System32\dnscmd.exe', '_msdcs.bar.domain.local', '/RecordAdd', 'bar.domain.local', 'alias.foo.bar.domain.local.', 'CNAME', 'host.foo.bar.domain.local').once.returns([nil, nil, nil])
```

Not sure about